Author: Tom Parker-Shemilt <palfrey@tevp.net>
Summary: Enables non-interactive builds

installnetpbm.pl defaults to wanting keyboard interactivity, which we don't want!
----
--- a/buildtools/installnetpbm.pl.orig
+++ b/buildtools/installnetpbm.pl
@@ -186,2 +186,3 @@
         my $default = "$prefix/bin";
+        return $default;
 
@@ -244,2 +245,3 @@
         my $default = "$prefix/lib";
+        return $default;
 
@@ -508,2 +510,3 @@
         
+        return;
         my $done;
@@ -575,2 +578,3 @@
         my $default = "$prefix/lib";
+        return $default;
 
@@ -638,2 +642,3 @@
         my $default = "$prefix/lib";
+        return $default;
 
@@ -675,2 +680,3 @@
         my $default = "$prefix/include";
+        return $default;
 
@@ -769,2 +775,3 @@
         my $default = "$prefix/man";
+        return "$prefix/share/man";
 
@@ -810,2 +817,3 @@
 sub tryToCreateManwebConf($) {
+    return;
 
@@ -1103,2 +1111,3 @@
         my $default = "$prefix/lib/pkgconfig";
+        return $default;
 
@@ -1184,5 +1193,5 @@
 
-my $pkgdir = getPkgdir();
+my $pkgdir = $ENV{'BUILDDIR'} . "/pkgdir";
 
-my $prefix = getPrefix();
+my $prefix = $ENV{'PKGDIR'} . "/usr";
 
